crates/solver/src/settlement.rs

```diff
@@ -478,9 +478,25 @@ impl Settlement {
             encoder: merged,
             submitter: self.submitter,
             score: match (self.score, other.score) {
```

Comment:
ubernit: Maybe it would make sense to have "score merging" be a method on the `Score` type?